src/java2yaml/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace Microsoft.Content.Build.JavaYaml
+{
+    using System;

```

Comment:
docfx v2 has a conversion to put `using` inside `namespace`. It is different from the default behavior of Visual Studio. v3 has put it outside namespace for easier maintenance.